jdi-light/src/main/java/com/epam/jdi/light/elements/init/PageFactory.java

```diff
@@ -197,10 +198,10 @@ private static void initWebPage(WebPage webPage) {
     }
     //endregion
 
-    public static List<Class<?>> STOP_INIT_CLASSES = asList(
-        Object.class, WebPage.class, PageObject.class, UIElement.class,
+    public static ArrayList<Class<?>> STOP_INIT_CLASSES = new ArrayList<>(asList(
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6361799)